core/src/com/ericc/the/game/MainGame.java

```diff
@@ -16,10 +16,7 @@
 import com.ericc.the.game.map.Generator;
 import com.ericc.the.game.map.Map;
 import com.ericc.the.game.systems.logic.*;
-import com.ericc.the.game.systems.realtime.ScreenBoundariesGetterSystem;
-import com.ericc.the.game.systems.realtime.AnimationSystem;
-import com.ericc.the.game.systems.realtime.RenderSystem;
-import com.ericc.the.game.systems.realtime.TileChanger;
+import com.ericc.the.game.systems.realtime.*;
```

Comment:
Skoro potrzebujemy wszystkich systemów to równie dobrze można importować com.ericc.the.game.systems.*